pages/linux/rpm-ostree.md

```diff
@@ -0,0 +1,29 @@
+# rpm-ostree
+
+> A hybrid image/package system.
+> Manage ostree deployments, package layers, filesystem overlays, and boot configuration.
+> More information: <https://coreos.github.io/rpm-ostree/administrator-handbook/>.
+
+- Show rpm-ostree deployments in the order they'll appear in the bootloader:
+
+`rpm-ostree status`
+
+- Show package updates that will be performed on the next rpm-ostree upgrade:
+
+`rpm-ostree upgrade --preview`
+
+- Prepare a new ostree deployment with upgraded packages and reboot into the new deployment:
+
+`rpm-ostree upgrade --reboot`
+
+- Roll back and reboot into the previous ostree deployment:
+
+`rpm-ostree rollback --reboot`
+
+- Install a package into a new ostree deployment and apply that deployment to the live deployment:
+
+`rpm-ostree install <htop> --apply-live`
```

Comment:
```suggestion
`rpm-ostree install {{package}} --apply-live`
```

Reply:
Apparently I forgot tldr's syntax 🤦  thanks!

Reply:
Regarding examples, I saw this comment in the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming) and might be misunderstanding it:

> Use an actual value rather than a generic placeholder where appropriate. For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.

For this style recommendation, the `iostat` examples use `{{2}}` over `{{interval}}` because that's a reasonable default for all cases, whereas using `{{htop}}` in this example is a bit too specialized?

Reply:
Many of those rules emerged from what feels right. `seconds` may not be the best example for "appropriate", because you can hardly do something wrong with it. But this rule is mostly for examples like `{{time}}`, `{{date}}`, `{{version}}` where you don't really know what it should look like. 

Also it's clear that `{{seconds}}` is a placeholder, while `{{2}}` is a concrete value. `htop` is quite popular, but it could still be misinterpreted as placeholder. `{{package}}` on the other side is clearly a placeholder and you can hardly do anything wrong with the syntax.

Reply:
@marchersimon got it, thanks for the clarification!